### PR TITLE
Adds missing Teshari DSI

### PR DESCRIPTION
**Pull request by Cameron653**
https://i.imgur.com/UmrLEDT.png - (White)
https://i.imgur.com/vXGIFi3.png - (Light Blue)
https://i.imgur.com/H8NFgTD.png (Light Pink)
Operates the same as every other DSI.

### CODE REVIEW

File: code/modules/organs/robolimbs.dm [hunk 1]
```diff
@@ -42,7 +42,7 @@ var/const/standard_monitor_styles = "blank=ipc_blank;\
 	var/lifelike										 // If set, appears organic.
 	var/skin_tone										 // If set, applies skin tone rather than part color
 	var/blood_color = "#030303"
-	var/list/species_cannot_use = list(SPECIES_TESHARI, "Promethean") //VOREStation Add
+	var/list/species_cannot_use = list("Promethean")	 //VOREStation Edit
```
Comment (Arokha) [inline code/modules/organs/robolimbs.dm]: n o

You need to add a reverse of this to make this acceptable (as in, a species_allowed). I won't merge anything that enables every prosthetic for Teshari, they look totally insane.

Comment (Cameron653) [reply code/modules/organs/robolimbs.dm]: Oh, fair enough. Didn't think of them being able to use other prosthetic limbs.

Comment (Arokha) [reply code/modules/organs/robolimbs.dm]: You can just override it on at one with list("Promethean") instead if you don't want to do a whole allowed system.

**Comment by Cameron653** [comment]
Annnd fixed. Thanks for the easy fix!

**Pull request merged**